### PR TITLE
Run PowerShell in latest Kali

### PR DESCRIPTION
**Pull request by mrpnkt**

### CODE REVIEW

**Comment by iSazonov** [comment]
@mrpnkt Thanks for your contribution!

Markdown test failed - please correct.

**Review by mrpnkt — COMMENTED** [review]
error in code block language (markdown fail)

File: docs/installation/linux.md [hunk 1]
```diff
@@ -427,6 +427,19 @@ dpkg -i powershell_6.0.0-alpha.18-1ubuntu1.16.04.1_amd64.deb
 powershell
 ```
 
+### Run PowerShell in latest Kali (Kali GNU/Linux Rolling) without installing it
+
+```sh
+# Grab the latest App Image
+wget https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.18/PowerShell-6.0.0-alpha.18-x86_64.AppImag
```
Comment (TravisEz13) [inline docs/installation/linux.md]: You are missing an `e` in `AppImage`

**Review by TravisEz13 — CHANGES_REQUESTED** [review]
see code comments

File: docs/installation/linux.md [hunk 2]
```diff
@@ -427,6 +427,19 @@ dpkg -i powershell_6.0.0-alpha.18-1ubuntu1.16.04.1_amd64.deb
...
 ```
 
+### Run PowerShell in latest Kali (Kali GNU/Linux Rolling) without installing it
+
+```sh
+# Grab the latest App Image
+wget https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.18/PowerShell-6.0.0-alpha.18-x86_64.AppImage
+
+# Make executable
+chmod a+x PowerShell-6.0.0-alpha.18-x86_64.AppImag
```
Comment (TravisEz13) [inline docs/installation/linux.md]: Also here

File: docs/installation/linux.md [hunk 3]
```diff
@@ -427,6 +427,19 @@ dpkg -i powershell_6.0.0-alpha.18-1ubuntu1.16.04.1_amd64.deb
...
+
+# Start PowerShell
+./PowerShell-6.0.0-alpha.18-x86_64.AppImag
```
Comment (TravisEz13) [inline docs/installation/linux.md]: and here

**Review by TravisEz13 — CHANGES_REQUESTED** [review]

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**